python/templates/Constructor.jl.jinja2

```diff
@@ -2,19 +2,33 @@ include("{{ class.bare_type }}Struct.jl")
 {% for include in includes_jl['constructor'] %}
 {{ include }}
 {% endfor %}
+
+{% macro julia_parameters(params) %}
+{%- if params_jl -%}{
+{%- set comma = joiner(',') -%}
+{%- for par in params -%}
+{{ comma() }}Main.{{ par }}Struct
+{%- endfor -%}
+}{%- endif -%}
+{% endmacro %}
```

Comment:
If I remember correctly a very similar macro already exists in the `MutableStruct.jl.jinja2` template(?). Is it possible to combine both macros into one and place it into a `macros/julia.jinja2` file and simply import that here? It would probably have to take more arguments to deal with the different pre- and suffixes.